packages/zowe-explorer/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed an issue where the attributes were not shown for VSAM data sets when using the Show Attributes feature. [#3545](https://github.com/zowe/zowe-explorer-vscode/issues/3545)
 - Fixed an issue where right-click menu for data set shows wrong options after deleting multiple PDS members. [#3516](https://github.com/zowe/zowe-explorer-vscode/issues/3516)
 - Updated dependencies for technical currency purposes. [#3577](https://github.com/zowe/zowe-explorer-vscode/pull/3577)
+- Fixed an issue where searching for the root directory (`/`) under a profile caused the profile to show a placeholder rather than a list of the files and folders at that path.
```

Comment:
Is there a way to add an issue or PR link to this?

Reply:
Sorry Ana, I meant to add it at EoD yesterday but it slipped my mind. Addressed in latest commit 👍